ui/src/reducers/index.js

```diff
@@ -35,7 +43,7 @@ export default (state = initialState, action) => {
       };
     case APPS_SUCCESS:
       var fetchedApps = [];
-      action.result.forEach(app => {
+      action.result.forEach((app) => {
```

Comment:
What's the reason for this change?

Reply:
just the auto-linter, no functional change